### PR TITLE
Fixes #24 - Reworked report profiles and added reporting

### LINKED ISSUE #24: clarification of the reporting is needed
the topic could also be placed in the user documentation, we have to talk about it. however, there is a lot of confusion about the interface, which we should try to eliminate.
mainly these words should be clarified:
- First Solution
- Reopen
- Moved in
- Moved out
- Backlog
- Difference between Communication & Create-Channel-Selection
- what does the selection mean (for example when selecting Ticket count: Backlog, Create Channel: Phone in and Communication: Email out)

### PR DESCRIPTION
**Pull request by ralf401**

### CODE REVIEW

**Comment by MrGeneration** [comment]
Reporting is not an administrational thing but a agent thing to do with fitting permissions.
I personally believe the reporting part is wrong in the admin docs.

**Comment by ralf401** [comment]
> Reporting is not an administrational thing but a agent thing to do with fitting permissions. I personally believe the reporting part is wrong in the admin docs.

I agree that it is not an ideal location. But in my eyes, the "standard" agent has nothing to do with it, too. So I would leave it until the documentation is perhaps completely rebuilt in the future.

File: manage/report-profiles.rst [hunk 1]
```diff
@@ -1,45 +1,90 @@
...
+Report Profiles and Reporting
+*****************************
 
-Report profiles are used to restrict / filter report-results. The idea of the
+.. Reporting included here because it fits more to the administrator role than
+.. the user role. /rsc
+
+The reporting is useful to view statistics, get an overview of the number of
+tickets (e.g. tickets of a specific customer) and to download ticket data from
+Zammad. In the following section you will find a description of how to create
```
Comment (dvuckovic) [inline manage/report-profiles.rst:9]: `a description of how` => `an instruction on how`

File: manage/report-profiles.rst [hunk 2]
```diff
@@ -1,45 +1,90 @@
...
+and change report profiles. In another section you can find information
```
Comment (dvuckovic) [inline manage/report-profiles.rst:10]: `In another` => `In a separate`

File: manage/report-profiles.rst [hunk 3]
```diff
@@ -1,45 +1,90 @@
...
+on basic usage of the reporting UI itself. If the reporting options in Zammad
+are not enough for you, we added a section about external reporting tools, you
```
Comment (dvuckovic) [inline manage/report-profiles.rst:12]: `,` => ❌

File: manage/report-profiles.rst [hunk 4]
```diff
@@ -1,45 +1,90 @@
...
+can use.
+
+To create and edit report profiles, ``admin.report_profile`` permissions are
```
Comment (dvuckovic) [inline manage/report-profiles.rst:15]: `are` => `is`

File: manage/report-profiles.rst [hunk 5]
```diff
@@ -1,45 +1,90 @@
...
+
+.. include:: /misc/object-conditions/conditioning-depth-hint.include.rst
+
+The edit dialog looks like this:
 
 .. figure:: /images/manage/report-profiles/profile-filtering-for-specific-organization-by-created-at-within-last-month.png
+    :alt: Screenshot shows creation of report profile
 
 This example shows the statistics of all tickets of the organization
 "Awesome Customer" that were created in the last month.
```
Comment (dvuckovic) [inline manage/report-profiles.rst:40]: `"Awesome Customer"` => `"Awesome Customer Inc."`

File: manage/report-profiles.rst [hunk 6]
```diff
@@ -1,45 +1,90 @@
...
 
-The filters can be combined with each other as desired. The filters build on
-each other, which means that they are further restricted per additional
-attribute.
+All configured report profiles are displayed in the reporting area and you can
+switch between them with one click. Have a look in the next section to learn
```
Comment (dvuckovic) [inline manage/report-profiles.rst:43]: `in` => `at`

File: manage/report-profiles.rst [hunk 7]
```diff
@@ -1,45 +1,90 @@
...
+.. figure:: /images/manage/report-profiles/reporting-sections.png
+    :alt: Screenshot showing different sections in the reporting screen
 
-.. figure:: /images/manage/report-profiles/reporting-time-frame.png
+1. **Additional filtering** based on the selected profile (see 2). You can
+   filter by status ("Ticket Count"), "Creation Channels" and "Communication"
+   types based on your channels.
+2. **Profile switcher**: here you can easily switch between the different
+   profiles, which were created in the admin panel under "Report Profiles".
+   The shown tickets and numbers are always limited to the current profile,
```
Comment (dvuckovic) [inline manage/report-profiles.rst:68]: `,` => ❌

File: manage/report-profiles.rst [hunk 8]
```diff
@@ -1,45 +1,90 @@
...
+   you have selected here.
+3. **Time intervall/period switcher** and **graph** area: here you can define
```
Comment (dvuckovic) [inline manage/report-profiles.rst:70]: `Time intervall` => `Time interval`

File: manage/report-profiles.rst [hunk 9]
```diff
@@ -1,45 +1,90 @@
...
+   the intervall you want to see (e.g. "Month") as well as the time period (e.g.
```
Comment (dvuckovic) [inline manage/report-profiles.rst:71]: `intervall` => `interval`

File: manage/report-profiles.rst [hunk 10]
```diff
@@ -1,45 +1,90 @@
...
+   "Jul").
+4. **Preview and download** section: here you can find a preview of tickets and
+   a download button based on the report profile and your filtering. The
+   download feature provides the the tickets in a ``.xlsx`` spreadsheet.
```
Comment (dvuckovic) [inline manage/report-profiles.rst:75]: `the the` => `the`

**Review by dvuckovic — APPROVED** [review]

**Pull request merged**